src/agent/onefuzz-supervisor/src/config.rs

```diff
@@ -103,17 +107,19 @@ impl StaticConfig {
             None
         };
 
-        let instrumentation_key = if let Ok(key) = std::env::var("ONEFUZZ_INSTRUMENTATION_KEY") {
-            Some(Uuid::parse_str(&key)?)
-        } else {
-            None
-        };
+        let instance_telemetry_key =
```

Comment:
note: this is currently unexercised functionality intended to enable the `docker container` scenario.